user/models.py

```diff
@@ -57,3 +57,7 @@ class User(AbstractBaseUser, PermissionsMixin):
     EMAIL_FIELD = "email"
     REQUIRED_FIELDS = ['username']
 
+class ConnectionModel(models.Model):
+    follower = models.ForeignKey(User, related_name='follower', on_delete=models.CASCADE)
+    following = models.ForeignKey(User, related_name='following', on_delete=models.CASCADE)
+    date_creagted = models.DateTimeField(auto_now_add=True) #typo
```

Comment:
- typo
- `DateTime`型を採用するのであれば以下の変更提案
```suggestion
    created_at = models.DateTimeField(auto_now_add=True)
```

Reply:
カラム名を`created_at`に変更しました。

参考記事: [Django3でテーブルのカラム(フィールド)名を変更する](https://qiita.com/holly0819/items/41f01096f59416b0d52b)